ksqldb-engine/src/main/java/io/confluent/ksql/schema/ksql/inference/SchemaRegisterInjector.java

```diff
@@ -118,6 +121,12 @@ private void registerSchema(
       } catch (IOException | RestClientException e) {
         throw new KsqlStatementException("Could not register schema for topic.", statementText, e);
       }
+    } else {
+      throw new KsqlStatementException(
```

Comment:
@agavra would it actually be possible for you to throw a nested `KsqlSchemaRegistryNotConfiguredException` here? In cloud, we want to throw a very specific error message related to this and we're passing in a cloud specific error message module in the cloud image.

https://github.com/confluentinc/ksql/blob/master/ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/Errors.java#L236
https://github.com/confluentinc/ksql/blob/master/ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/Errors.java#L218